### ISSUE TITLE
Playlists are not generated

### ISSUE THREAD

**Issue by tony-jeffreys** [epsil/spotgen#25]
It looks like playlists are no longer generated when the "Create Playlist" button is clicked.

**Comment by jolietnick** [comment]
Yep. It first goes to "https://epsil.github.io/spotgen/#error=unsupported_response_type", then it goes to "INVALID_CLIENT: Invalid redirect URI" if you click on create a second time

**Comment by jolietnick** [comment]
They changed the API so if nobody intervenes, the frontend won't work anymore
